Bio/GenBank/Scanner.py

```diff
@@ -733,7 +756,10 @@ def _feed_first_line_patents_kipo(self, consumer, line):
         # e.g. ID   DI500001       STANDARD;      PRT;   111 AA.
         #
         # This follows the style of _feed_first_line_old
-        assert line[: self.HEADER_WIDTH].rstrip() == "ID"
+        if line[: self.HEADER_WIDTH].rstrip() != "ID":
+            raise ValueError(
+                f"Expected first {self.HEADER_WIDTH} letters in 'line' to contain 'ID'."
+            )
```

Comment:
Rather than 'line' which is just the local variable name, how about "... in first line to contain ID"?

Likewise in the next few asserts. It is hard to tell from the diff alone, but you could also say "LOCUS line" etc depending in GenBank or EMBL.